syft/frameworks/torch/he/fv/util/operations.py

```diff
@@ -57,8 +57,21 @@ def invert_mod(value, modulus):
         return gcd_tuple[1]
 
 
-def poly_add_mod(op1, op2, coeff_mod):
-    """Add two polynomails and modulo every coefficient with coeff_mod.
+def poly_add(op1, op2, poly_mod):
+    """return addition of two polynomials"""
+
+    # For non same size polynomials we have to shift the polynomials because numpy consider right
+    # side as lower order of polynomial and we consider right side as heigher order.
+    if len(op1) != poly_mod:
```

Comment:
maybe I'm missing something, but isn't poly_mod a list/a np.array/a polynomial?
is this check correct?

Reply:
poly_mod is an integer value thought the scheme, it gets converted to list representation only when required in the calculation.